dbt/models/staging/stg_emi_heures.sql

```diff
@@ -1,12 +1,13 @@
 select
     emi.emi_pph_id,
-    emi.emi_afi_id,
     emi.emi_ctr_id,
-    emi.emi_sme_annee,
-    sum(emi.emi_nb_heures_travail) as emi_nb_heures_travail
-from {{ ref('fluxIAE_EtatMensuelIndiv_v2') }} as emi
+    min(subquery.date_emi) as date_recrutement_reelle
+from {{ ref("fluxIAE_EtatMensuelIndiv_v2") }} as emi
+left join {{ ref("fluxIAE_EtatMensuelIndiv_v2") }} as subquery
+    on emi.emi_ctr_id = subquery.emi_ctr_id
+where
+    emi.emi_ctr_id in (emi.emi_ctr_id)
+    and subquery.emi_nb_heures_travail > 0
```

Comment:
La clause `emi.emi_ctr_id in (emi.emi_ctr_id)` me semble être toujours vraie, et on est d'accord que `subquery.emi_nb_heures_travail > 0` fait que le `left join` n'en est plus vraiment un, non ? :thinking:

Reply:
Le `where mi.emi_ctr_id in (emi.emi_ctr_id)` est la pour avoir la liste de tous les `emi_ctr_id`
Donc c'est bien un left join